### ISSUE TITLE
To-do list for reportSurvey_2207RL

### ISSUE THREAD

**Issue by jrenfree** [kstierhoff/estimATM#4]
- [ ] In the transect summaries, it now uses 'nasc' to compute the number of transects, however when that variable is made in process_NASC it is not correctly removing the transect parts.
  - [x] #12
- [ ] Reprocess Lasker data:
  - [ ] Using original EV files (in V12), replace ECS file with corrected ones containing accurate calibration parameters
  - [ ] Update paths
  - [ ] Update CPS filter operands
  - [ ] Re-export integration
  - [ ] Re-run CSV files through R script to apply range mask
- [ ] Reprocess Lisa Marie data after updating Sv-differencing threshold (looks to be using 120 kHz ranges, not 200 kHz)
  - Putting this on hold until we can confirm if it is an issue
- [ ] Reprocess Saildrone data after updating Sv-difference threshold (looks to be using 120-kHz ranges, not 200 kHz)
  - Putting this on hold until we can confirm if it is an issue
- [ ] Incorporate LBC NASC data into survey report (waiting for results from Scott)
  - Code has been added to reportSurvey to plot the data, now just awaiting results
- [ ] Incorporate Saildrone NASC data into survey report (waiting for results from Alice)
  - Code in reportSurvey should plot SD NASC data, now just awaiting processed data
- [ ] Create habitat map using proximal locations to survey vessels
  - Code and map have been made, but still waiting on CSV files for LBC and SD
- [x] Locate Lasker's 18-kHz calibration data
- [x] Remove pre-survey habitat map and indicate that the transects were planned based on historical observations
- [x] Update Lisa Marie processing steps in report, as it now includes 200 kHz data
- [x] Re-process Transect 178 from Lisa Marie using R script to include putative herring
- [x] Create a figure of the transects surveyed by each vessel, then incorporate figure into the survey report
  - The file Doc\plotSurvey_2207RL.Rmd contains a lot of plotting code that Kevin put together ad-hoc, and contains a section for plotting nav data from multiple vessels
  - Alternatively, can use integrated NASC data to plot "nav" data from just the transects, so that it doesn't include transits, which might be better
- [x] Update calibration files used for tables
  - Currently have it so that the XML files are read but then the ECS files are read to overwrite the results parameters. However will need to add specific code to merge the cal results from both Saildrones into a single table
- [x] plot_purseSeine_xxxx saved a file called 'nasc_summ_tx_ns.Rdata', which gets used when summarizing nearshore transects in the manuscript text. Will need to either fix this script or add code to derive this variable
- [x] Incorporate Shimada CTD casts into survey report
- [x] Incorporate LM NASC data into survey report
  - [x] Process LM data through R script
- [x] Incorporate LM seine data into survey report
- [x] Update personnel for purse-seine sampling
  - [x] Lisa Marie
  - [x] Long Beach Carnage
- [x] Re-enable code chunk for displaying Saildrone calibration results
- [x] Obtain and organize SCS data for processing in surveyReport
- [x] Add Lisa Marie (and Saildrone?) data to transects where they surveyed missed Lasker transects (i.e., treat them as core area)
  - Need to add catch data and NASC data (does not have CUFES)
- [x] Need to generate Saildrone ECS files that use Lasker CTD/UCTD casts. Can either re-process Lasker (u)CTD data using the template Saildrone files, or write a script to batch process the Lasker ECS files but replacing them with Saildrone calibration parameters
